subsys/bluetooth/host/Kconfig.gatt

```
@@ -284,6 +284,31 @@ config DEVICE_NAME_GATT_WRITABLE_AUTHEN
 
 endif #BT_DEVICE_NAME_GATT_WRITABLE
 
+config BT_DEVICE_APPEARANCE_GATT_WRITABLE
+	bool "Allow to set GAP Appearance by remote GATT clients"
+	depends on BT_DEVICE_APPEARANCE_DYNAMIC
+	default y
+	help
```

```suggestion
	depends on BT_DEVICE_APPEARANCE_DYNAMIC
	help
```

I don't think this should be the default

---

ok, I wanted to keep this consistent with name, but I'm fine with that

---

Yeah, making the GAP GATT attributes similar is something that definitely makes sense. Maybe the name needs to match this instead, but that's outside the scope of this :)